### PR TITLE
feat(std/http): Validate cookie path value

### PR DESCRIPTION
**Pull request by getspooky**

### CODE REVIEW

File: std/http/cookie.ts [hunk 1]
```diff
@@ -92,6 +93,26 @@ function validateCookieName(name: string | undefined | null): void {
...
+ * @param path Path value.
+ */
+function validatePath(path: string | null): void {
+  if (path == null) {
+    return;
+  }
+  for (let i = 0; i < path.length; i++) {
+    const c = path.charAt(i);
+    if (
+      c < String.fromCharCode(0x20) || c > String.fromCharCode(0x7E) || c == ";"
```
Comment (nayeemrmn) [inline std/http/cookie.ts:107]: Reference?

Comment (getspooky) [reply std/http/cookie.ts:107]: https://tools.ietf.org/html/rfc6265#section-4.1.2.4

Comment (nayeemrmn) [reply std/http/cookie.ts:107]: Okay, found it in https://tools.ietf.org/html/rfc6265#section-4.1.1:
```
path-value        = <any CHAR except CTLs or ";">
```
(where a `CHAR` is an ASCII character).

**Review by nayeemrmn — APPROVED** [review]
LGTM, thanks.

File: std/http/cookie.ts (hunk 1 above)
Comment (bartlomieju) [reply std/http/cookie.ts:107]: Please add this reference as a comment

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @getspooky

**Pull request merged**